chart/values.yaml

```diff
@@ -637,3 +633,27 @@ obs:
     tolerations: []
     # -- Set PriorityClass, overrides global
     priorityClassName: ""
+  # Eventing component which enables or disables eventing-related components.
```

Comment:
This comment should mention how to disable stats

Reply:
We can mention that in the comment.

Reply:
done